src/dinero.js

```diff
@@ -574,8 +574,10 @@ const Dinero = options => {
      */
     toRoundedUnit(precision) {
       const factor = Math.pow(10, precision)
+
       return calculator.divide(
-        Math.round(calculator.multiply(this.toUnit(), factor)),
+        Math.sign(this.toUnit()) *
```

Comment:
I was just thinking this should be done with `Calculator#multiply` as well. Will be more future-proof when support for big integers is added.

```
calculator.multiply(Math.sign(this.toUnit()), Math.round(Math.abs(calculator.multiply(this.toUnit(), factor))))
```